tests/torch/test_models/swin.py

```diff
@@ -272,7 +272,7 @@ def __init__(
     def forward(self, x):
         H, W = self.input_resolution
         B, L, C = x.shape
-        assert L == H * W, "input feature has wrong size"
+        assert H * W == L, "input feature has wrong size"
```

Comment:
What's the rule in such comparisons?

Reply:
https://docs.astral.sh/ruff/rules/yoda-conditions/

Reply:
Thanks!